lib/Cro/WebApp/Template/ASTBuilder.pm6

```diff
@@ -39,7 +39,7 @@ class Cro::WebApp::Template::ASTBuilder {
     }
 
     method sigil-tag:sym<variable>($/) {
-        make escape(VariableAccess.new(name => '$' ~ $<identifier>, attribute => $<attribute>));
+        make escape(VariableAccess.new(name => '$' ~ $<identifier>, attribute => $<deref>));
```

Comment:
The point here was to write something like:

```
        my $derefer = $<deref> ?? $<deref>.ast !! { $_ };
         make escape($derefer(VariableAccess.new(name => '$' ~ $<identifier>)));
```

And not to have the `attribute` property on the `VariableAccess` node at all. The reason being that in the future `deref` will support a variety of different things, including chained (like `<$foo.bar.baz>`), and we can re-use that logic for `<.foo.bar>` and `<$baz.foo.bar>` cases, and others too.